### ISSUE TITLE
Disabling hold timing violations fixing for inputs and outputs

### ISSUE THREAD

**Issue by donn** [The-OpenROAD-Project/OpenLane#860]
Reported by @shalan 

### Prompt
The ECO flow currently attempts to fix hold timing violations on inputs and outputs, which won't really work if there are timing constraints.

### Proposal
Show how to use the ECO flow to disable hold timing violation fixing for IOs. If the feature doesn't exist, it should be surfaced.

**Comment by msaligane** [comment]
Yes, we identified that. we will add an option to skip pin-to-reg fixes.

**Comment by msaligane** [comment]
#863 should fix this.
